integration_tests/e2e/workAndSkillsPage.cy.ts

```diff
@@ -181,7 +181,7 @@ context('Work and skills page', () => {
           workAndSkillsPage
             .GoalsInfo()
             .contains(
-              'The prisoner education team set these goals using Virtual Campus. They do not include sentence plan targets. Contact the local education team to find out more.',
+              'The careers, information, advice and guidance (CIAG) team set these goals using Virtual Campus (VC2). They do not include sentence plan targets.',
```

Comment:
It might be worth reverting your change to this file and then rebasing.
My last PR (that moves this file) has been merged I think so you'll end up with a merge conflict; plus I'm currently working on the next part of the cypress tests for the W&S tab, so might be worth backing away from any changes here for the time being?